src/features/workflow/modal/DeployModal.tsx

```diff
@@ -178,12 +179,13 @@ const DeployModal: React.FC = () => {
     buttonType = 'light'
   }
 
+  const isDeploying = () => {
+    return deploying || deployStatus === 'deployed' || deployStatus === 'deploying' || deployStatus === 'failed'
+  }
+
   return (
-    <div style={{ width: 440, height: 440 }}>
-      <div className={classNames('absolute w-full h-full bg-white p-8 transition-all duration-300 bg-white p-8 text-left text-black flex flex-col z-10', {
-        'left-0': !deploying,
-        '-left-full': deploying
-      })}>
+    <div style={{ width: 440 }}>
+      {!isDeploying() && <div className={'w-full h-full bg-white p-8 transition-all duration-300 bg-white p-8 text-left text-black flex flex-col z-10'}>
```

Comment:
it looks like we are losing a transition moment here with this change.

If you look at the deploy screens in `master`, the screen swipes to the left when you click "run deploy", on this branch we don't have a transition.